src/System.IO.Compression/src/System/IO/Compression/ZLibNative.cs

```diff
@@ -72,7 +72,7 @@ public enum CompressionLevel : int
         {
             NoCompression = 0,
             BestSpeed = 1,
-            BestCompression = 9,    // Refer to "How to choose a compression level" above.
+            BestCompression = 6,    // Refer to "How to choose a compression level" above.
```

Comment:
> Does that look right to you?

This mapping looks perfect to me.

>  ZlibNative.CompressionLevel.BestCompression will be removed since we won't be using it anywhere.

Yep - no reason to leave it in if it isn't used.  

One additional thing to update is the native "PAL" layer that ensures our hard-coded values match the zlib we are compiling against:

https://github.com/dotnet/corefx/blob/master/src/Native/System.IO.Compression.Native/pal_zlib.cpp#L21-L23